appveyor.yml

```diff
@@ -31,6 +31,7 @@ test:
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
     - '**\Sarif.Viewer.VisualStudio.UnitTests.dll'
+    - '**\Microsoft.CodeAnalysis.SarifCli.FunctionalTests.dll'
```

Comment:
>Microsoft.CodeAnalysis [](start = 10, length = 22)

Why is this naming convention different? Should be SarifCli.FunctionalTests.dll.